src/event.c

```diff
@@ -224,13 +221,8 @@ void xmpp_run_once(xmpp_ctx_t *ctx, unsigned long timeout)
                 FD_SET(conn->sock, &wfds);
             else {
                 strophe_info(ctx, "xmpp", "Connection attempt timed out.");
-                ret = _connect_next(conn);
-                if (ret != 0) {
-                    conn->error = ETIMEDOUT;
```

Comment:
as written, this error is no longer produced and will always end up as XMPP_EINT instead